modules/integration/tests-integration/lazy-loading-test/src/test/java/org/wso2/appserver/integration/lazy/loading/util/WebAppStatus.java

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
Class that are in the test-common is to  create the  the  web application(.war file), its under common-artefacts.  class in-side the   org.wso2.appserver.integration.lazy.loading.util  are  use by test classes.